src/App.js

```diff
@@ -1,9 +1,14 @@
 import "./App.css";
-
+import "./styles/common.css";
+import "./styles/layouts/scrollbar.css";
+import "./styles/utils/variables.css";
+import { PageRoutes } from "./routes/PageRoutes";
 function App() {
   return (
     <>
-      <div className="App">AgroPlay</div>
+      <div className="App">
+        <PageRoutes />
+      </div>
     </>
```

Comment:
Use can this fragment <>, as your are returning only one div

Reply:
made changes in new commit, Thanks